utils/github_adapter.py

```diff
@@ -0,0 +1,232 @@
+            'X-GitHub-Api-Version': '2022-11-28'
+        }
+
+        async with httpx.AsyncClient() as client:
+            try:
+                # Make the request to the GitHub API
+                response = await client.get(github_api_url, headers=headers)
+                # Check if the request was successful
+                if response.status_code == 200:
+                    # Return the response from the GitHub API
+                    response,code = response.json(),200
+                    
+            except :
+                response = [],400
+                
+            return response,code
+    
+    async def createComment(self, owner, repo, issue_number, markdown_dict):
+        token = await GithubAPI().authenticate_app_as_installation(repo_owner=owner)
+
+        url = f'https://api.github.com/repos/{owner}/{repo}/issues/{issue_number}/comments'
+        headers = {
+            'Accept': 'application/vnd.github+json',
+            'Authorization': f'Bearer {token}',
+            'X-GitHub-Api-Version': '2022-11-28'
+        }
+        data = {
+            'body': f'{TicketFeedbackHandler.feedBackMessageCreator(markdown_dict=markdown_dict)}'
+        }
+
+        async with aiohttp.ClientSession() as session:
+            async with session.post(url, headers=headers, json=data) as response:
+                if response.status == 201:
+                    print('Comment created successfully.')
+                    return await response.json()
+                else:
+                    print(f'Error creating comment. Status code: {response.status}', sys.stderr)
+                    response_text = await response.text()
+                    print(f'Response body: {response_text}', file=sys.stderr)
+    
+    async def updateComment(self, owner, repo, comment_id, markdown_dict):
+        token = await GithubAPI().authenticate_app_as_installation(repo_owner=owner)
+        url = f'https://api.github.com/repos/{owner}/{repo}/issues/comments/{comment_id}'
+        headers = {
+            'Accept': 'application/vnd.github+json',
+            'Authorization': f'Bearer {token}',
+            'X-GitHub-Api-Version': '2022-11-28'
+        }
+        data = {
+            'body': f'{TicketFeedbackHandler.feedBackMessageCreator(markdown_dict)}'
+        }
+
+        async with aiohttp.ClientSession() as session:
+            async with session.patch(url, headers=headers, json=data) as response:
+                if response.status == 200:
+                    print('Comment updated successfully.')
+                    return await response.json()
+                else:
+                    print(f'Error updating comment. Status code: {response.status}', file=sys.stderr)
+                    response_text = await response.text()
+                    print(f'Response body: {response_text}',file=sys.stderr)
+    
+    
+    async def deleteComment(self, owner, repo, comment_id):
```

Comment:
let's have consistent naming for all variables and functions adhering to the [style guide](https://peps.python.org/pep-0008/#function-and-variable-names)